src/gpl/test/BUILD

```diff
@@ -0,0 +1,60 @@
+load("//test:regression.bzl", "regression_test")
+
+TESTS = [
+"ar01",
+"ar02",
+"clust01",
+"clust02",
+"clust03",
+"cluster_place01",
+"convergence01",
+"core01",
+"density01",
+"diverge01",
+"error01",
+"incremental01",
+"incremental02",
+"nograd01",
+"simple01",
+"simple01-obs",
+"simple01-rd",
+"simple01-ref",
+"simple01-skip-io",
+"simple01-td",
+"simple01-td-tune",
+"simple01-uniform",
+"simple02",
+"simple02-rd",
+"simple03",
+"simple03-rd",
+"simple04",
+"simple04-rd",
+"simple05",
+"simple06",
+"simple07",
+"simple08",
+"simple09",
+"simple10",
+]
+
+[
+    regression_test(
+        name = name,
+        data =
+            # overly broad glob, could be refined later, but
+            # symlinks are cheap and OpenROAD binary changes, the common
+            # use case is that all tests have to be re-run.
+            glob(
+                ["**/*"],
+                exclude = [
+                    test + "." + ext
+                    for test in TESTS
+                    for ext in [
+                        "tcl",
+                        "py",
+                    ]
+                ],
+            ),
```

Comment:
Instead of the glob you might want to put them into a filegroup and references them in the data dep.

Reply:
This is used once, so I would still use a glob, it would simply add a level of indirection. Is it a matter of syntactic sugar or is it in some way different?

Reply:
It would give a name to this set of files (as we might need it in other places as well, and it also allows to then compose things from smaller parts, such as pdk data and tcl scripts)

Reply:
fixed